src/features/about/about.data.ts

```diff
@@ -50,32 +52,63 @@ const awsCloudDeveloper: AboutInfo[] = angularNodejsAwsFundamentals('aws cloud d
   return item;
 });
 
-const javaScript: (lang: string) => AboutInfo[] = () => [
-  {
-    id: 1,
-    title: 'For everyone',
-    info: 'Everyone can study at RS School, regardless of age, professional employment, or place of residence. However, you should have sufficient base knowledge before the program begins.',
-    icon: personIcon,
-  },
-  {
-    id: 2,
-    title: 'Worldwide mentors and trainers',
-    info: 'The Mentors and trainers of our school are front-end and javascript developers from different companies/countries. How to become a <a href="/courses#mentors-wanted">mentor</a>?',
-    icon: planetIcon,
-  },
-  {
-    id: 3,
-    title: 'Free education',
-    info: 'Feel the desire to share your experience and knowledge',
-    icon: giftIcon,
-  },
-  {
-    id: 4,
-    title: 'Certificate',
-    info: 'A certificate of successful completion of the course is issued to everybody who pass two stages of training.',
-    icon: awardIcon,
-  },
-];
+const javaScript: (lang: Language) => AboutInfo[] = (lang) => {
```

Comment:
Let's create 3 constants - for JS, JS RU and Pre School RU (for Pre school content may be the same for now).